scripts/test_call.py

```
@@ -0,0 +1,691 @@
+    "8":  "Local Hybrid Pipeline",
+    "9":  "ElevenLabs Agent",
+    "10": "Fully Local Pipeline",
+}
+
+IVR_NUMBER = "19257366718"
+
+# AMI credentials (from FreePBX manager.conf)
+AMI_HOST = "127.0.0.1"
+AMI_PORT = 5038
+AMI_USER = "OgtKe0U6pHl1"
+AMI_PASS = "Xr3CjJD0qVRR"
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to: "AMI_PASS"

(S105)